src/msg/async/EventKqueue.cc

```diff
@@ -99,22 +211,66 @@ int KqueueDriver::event_wait(vector<FiredFileEvent> &fired_events, struct timeva
   int retval, numevents = 0;
   struct timespec timeout;
 
+  ldout(cct,10) << __func__ << " kqfd = " << kqfd << dendl;
```

Comment:
@wjwithagen this line... we don't want to print level 10 log when entering event_wait each time